src/Database/PostgreSQL/Simple/Migration.hs

```diff
@@ -71,20 +73,23 @@ import           System.Directory                   (getDirectoryContents)
 -- a 'MigrationError' is returned.
 --
 -- It is recommended to wrap 'runMigration' inside a database transaction.
-runMigration :: MigrationContext -> IO (MigrationResult String)
-runMigration (MigrationContext cmd verbose con) = case cmd of
+runMigration
```

Comment:
You have modified the signature of an existing function which is against the maintainers wishes:

> That means no destructive signature changes to existing public functions or data-types - only extensions.

Reply:
> There seems to be quite a bit of destructive and unrelated changes.

I’d like to hear what do you mean specifically by a “destructive change”?

Reply:
Yes, and this is mentioned in the description. It’s not really *that destructive*. All the existing code is still compatible, except just in case you annotated `MigrationContext` type explicitly.